tests_login_with_email_address/test_visibility_of_elements.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+from assertpy import assert_that, soft_assertions
+
+
+def test_visibility(page):
+    with soft_assertions():
+
+        # page.goto("https://login.pracuj.pl/")
+
+        locators = page.locator("[data-test=\"input-email\"]")
+        locators2 = page.locator("[data-test=\"input-password\"]")
+        locators3 = page.locator("[data-test=\"button-login\"]")
+        locators4 = page.locator("[data-test=\"button-login-facebook\"]")
+        locators5 = page.locator("text= Nie masz konta? Zarejestruj się")
+        locators6 = page.locator("text=Jesteś pracodawcą? Zaloguj się do Pracuj.pl dla Firm launch")
+
+        if page.locator("[data-test=\"input-email\"]").is_visible():
```

Comment:
This entire statement is a little award because:

- you have already defined locator for this element so why not using `locator.is_visible()`?
- `assert_that` below and whole `else` statement should be also removed because it make no sense.

This entire part should be changed to:
`assert_that(email_input_loc.is_visible()).described_as("E-mail input is not visible").is_true()`
This is enough to check what you need. What's more is this case you can put locator just before this `assert_that` so they are in one place, easy to see and modify.